src/scss/06_components/navigation/global/_mmenu.scss

```diff
@@ -310,7 +310,8 @@ $-footerlink-target-size: 44px;
     padding: 0;
   }
 
-  .menu__link {
+  .menu__link,
+  .menu__nolink {
```

Comment:
Discussed this with Mikko, and the `menu__nolink` class should only have the padding and font-size styles, not any of the hover or focus styles (underline and outline).

Reply:
`.menu__nolink` is now as a own class.